### ISSUE TITLE
Error compiling xutl_offcrypto.pkb

### ISSUE THREAD

**Issue by aragorniiit** [mbleron/ExcelTable#23]
I'm getting below error compiling xutl_offcrypto.pkb due to [Error] PLS-00201 (1319: 14): PLS-00201: identifier 'FILE2BLOB' must be declared. Please advise.

Thanks
Kishore

**Comment by mbleron** [comment]
Looks like you had pulled the latest version of XUTL_OFFCRYPTO instead of the one bundled with ExcelTable.
The latest version was bugged because I left a piece of testing code which lead to the above error.
It is corrected now.
Thanks for raising this issue.

**Comment by aragorniiit** [comment]
I actually compiled only pl/sql code, forgot the java installation steps.
Let me try again after installation is complete.
